src/main/java/no/finn/unleash/FakeUnleash.java

```diff
@@ -1,158 +1,10 @@
 package no.finn.unleash;
 
-import java.util.ArrayList;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
-import java.util.function.BiFunction;
-import java.util.stream.Collectors;
-import no.finn.unleash.lang.Nullable;
-
-public final class FakeUnleash implements Unleash {
```

Comment:
You addressed an issue that I wanted fixed. This `FakeUnleash` is `final` while the `io.getunleash` one is not. I want to keep in NOT final and just wanted to unsure that was intentional.